File: builder/builder.py
class Computadora:
    def __init__(self):
        self.partes = []

    def agregar(self, parte):
        self.partes.append(parte)

class Builder:
    def construir(self):
        pass

class BuilderPC(Builder):
    def construir(self):
        pc = Computadora()
        pc.agregar("CPU")
        pc.agregar("RAM")
        return pc

File: builder/builder.java
package builder;

class Computadora {
    private String cpu;
    private String ram;

    public void setCpu(String cpu) {
        this.cpu = cpu;
    }

    public void setRam(String ram) {
        this.ram = ram;
    }
}

class BuilderPC {
    public Computadora build() {
        Computadora pc = new Computadora();
        pc.setCpu("Intel");
        pc.setRam("16GB");
        return pc;
    }
}
